Resources/Prototypes/Species/vox.yml

```diff
@@ -0,0 +1,138 @@
+- type: species
+  id: Vox
+  name: Vox
+  roundStart: true
+  prototype: MobVox
+  sprites: MobVoxSprites
+  markingLimits: MobVoxMarkingLimits
+  dollPrototype: MobVoxDummy
+  skinColoration: Hues
+
+- type: speciesBaseSprites
+  id: MobVoxSprites
+  sprites:
+    Head: MobVoxHead
+    Hair: MobHumanoidAnyMarking
+    FacialHair: MobHumanoidAnyMarking
+    Chest: MobVoxTorso
+    Eyes: MobVoxEyes
+    LArm: MobVoxLArm
+    RArm: MobVoxRArm
+    LHand: MobVoxLHand
+    RHand: MobVoxRHand
+    LLeg: MobVoxLLeg
+    RLeg: MobVoxRLeg
+    LFoot: MobVoxLFoot
+    RFoot: MobVoxRFoot
```

Comment:
Wait so how would all this work with, say, surgery, if you swapped someone's limbs out?

Reply:
you would have to do a call that adds a 'custom base sprite' to a humanoid component - so if you wanted to swap somebody's limb with a lizard limb or a slimeperson limb, you'd just add the base sprite to the person, and the client would deal with it